examples/io/plot_objects_from_arrays.py

```diff
@@ -60,6 +63,7 @@
 ###############################################################################
 # EpochsArray
 
+# This is used to identify the events
```

Comment:
this is not super-clear, maybe better to say that events array is created where the first row defines event time in samples and the last column specifies event identity?